eng/pipelines/libraries/build-job.yml

```diff
@@ -110,7 +110,19 @@ jobs:
             inputs:
               sourceFolder: $(Build.SourcesDirectory)/artifacts/bin/runtime
               targetFolder: $(Build.ArtifactStagingDirectory)/artifacts/bin/runtime
-          
+
+          - task: CopyFiles@2
```

Comment:
I guess this will need to be rebased if: https://github.com/dotnet/runtime/pull/494 is merged first.

Reply:
Yeah, I think the MSBuild changes are probably going to be the bigger conflicts though. I'm not expecting to be able to merge them in particularly quick succession. 😄